www/js/pages/trade/components/payout/payout.controller.js

```diff
@@ -62,7 +62,7 @@
         };
 
         vm.stopLongPress = () => {
-            vm.proposal.amount = vm.amount;
+            vm.proposal.amount = typeof vm.amount === "string" ? Number(vm.amount) : vm.amount;
```

Comment:
@nazaninreihani Also here
```suggestion
            vm.proposal.amount = +vm.amount;
```